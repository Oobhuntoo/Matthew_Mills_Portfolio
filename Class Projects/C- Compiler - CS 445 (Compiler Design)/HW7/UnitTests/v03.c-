main()
{
   loop i=0..10:-1 do output(i);
   output(666);
}
