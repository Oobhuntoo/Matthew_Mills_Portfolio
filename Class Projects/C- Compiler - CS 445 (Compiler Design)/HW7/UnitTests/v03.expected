Loading file: v03.tm
666 
Status: Halted
Bye.
